src/software/simulation/simulator.cpp

```diff
@@ -23,6 +23,8 @@ Simulator::Simulator(const Field& field, const Duration& physics_time_step)
 Simulator::Simulator(const Field& field, double ball_restitution,
                      double ball_linear_damping, const Duration& physics_time_step)
     : physics_world(field, ball_restitution, ball_linear_damping),
+      yellow_team_defending_side(FieldSide::NEG_X),
+      blue_team_defending_side(FieldSide::NEG_X),
```

Comment:
Do the `team_defending_side` variables need to default to opposite sides (i.e should one of these be `FieldSide::POS_X`?

Reply:
Not necessarily. Any AI running on the network should be sending a value, and SimulatedTests only control a single team (and explicitly set the side), so it doesn't matter much there.

I don't think there's a huge benefit to having the default be either way, since if the AI using the sim doesn't match the defending side value, robots won't move properly, so they should always be setting it explicitly when running. Totally fine changing it if you think that's better though.

Reply:
That makes sense, all good